examples/ghost/src/generated/ghost-nexus.ts

```diff
@@ -58,43 +58,43 @@ export type NexusGenAllTypes = NexusGenRootTypes & NexusGenScalars
 export interface NexusGenFieldTypes {
   Post: {
     // field return type
-    html: string | null // String
-    slug: string | null // String
-    title: string | null // String
-    uuid: string | null // String
+    html: string // String!
+    slug: string // String!
+    title: string // String!
+    uuid: string // String!
   }
   Query: {
     // field return type
     me: NexusGenRootTypes['User'] | null // User
-    postById: NexusGenRootTypes['Post'] | null // Post
-    userById: NexusGenRootTypes['User'] | null // User
+    postById: NexusGenRootTypes['Post'] // Post!
+    userById: NexusGenRootTypes['User'] // User!
   }
   User: {
     // field return type
     accessibility: string | null // String
     bio: string | null // String
     coverImage: string | null // String
-    createdAt: NexusGenScalars['Date'] | null // Date
-    createdBy: NexusGenRootTypes['User'] | null // User
-    email: string | null // String
```

Comment:
Yes and no. This example, like others, had their typegen file changed when we switch the nonNullDefaults to be false by default. I made this example use nonNullDefaults: { output: true } again, so that the usages of nullable() make sense (just like we were doing nullable: true before).

After making output types nullable by defaults, there were many places where we were doing nullable: true although the field was already nullable. I did not create these examples but wanted to reflect again what was initially meant to be done